src/translations/fr-CA.json

```diff
@@ -42,8 +43,8 @@
 	"orc-shared.scopeChangeWithUnsavedDataConfirmation": "Une ou plusieurs entités ouvertes seront fermées, entraînant la perte de données non-sauvegardées. Voulez-vous vraiment changer de Scope?",
 	"orc-shared.scopeChangeWithUnsavedDataTitle": "Confirmation du changement de Scope sans sauvegarde",
 	"orc-shared.scopeFilterPlaceholder": "Entrez un nom de Scope",
-	"orc-shared.showFewerLanguages": "Afficher moins de langues",
-	"orc-shared.showMoreLanguages": "Afficher plus de langues",
+	"orc-shared.showFewerLanguages": "Afficher plus de langues",
```

Comment:
I'm, this needs to be reverted.... Fewer = mound, more is plus

Reply:
Corrected.
I don't understand why this has been changed.